srcpkgs/bastet/template

```diff
@@ -0,0 +1,29 @@
+# Template file for 'bastet'
+pkgname=bastet
+version=0.43.2
+revision=1
+archs="~*-musl"
+hostmakedepends="gcc"
+makedepends="boost-devel ncurses-devel"
+depends="boost ncurses"
+short_desc="Evil falling block game"
+maintainer="Alaisdair West <alaisdair.west2@outlook.com>"
+license="GPL-3.0-only"
+homepage="http://fph.altervista.org/prog/bastet.html"
+changelog="https://raw.githubusercontent.com/fph/bastet/master/NEWS"
+distfiles="https://github.com/fph/${pkgname}/archive/refs/tags/${version}.tar.gz"
+checksum=f219510afc1d83e4651fbffd5921b1e0b926d5311da4f8fa7df103dc7f2c403f
+nocheckperms=yes # For High Score file
+
+do_build() {
+	make
```

Comment:
You probably want to use `build_style=gnu-makefile`

Reply:
Unfortunately the makefile provided by bastet does not seem to work with that